plugins/postcss-color-mix-function/test/basic.expect.css

```diff
@@ -40,11 +40,11 @@
 }
 
 .spec-example-6 {
-	color-1: rgb(166, 158, 255);
-	color-1: color(display-p3 0.64126 0.60728 1);
+	color-1: rgb(174, 138, 255);
+	color-1: color(display-p3 0.66273 0.5437 0.98509);
 	/* TODO : https://github.com/w3c/csswg-drafts/issues/8609 */
-	color-2: rgb(116, 163, 255);
-	color-2: color(display-p3 0.49247 0.63302 1);
+	color-2: rgb(119, 163, 255);
```

Comment:
Slightly different but close enough to not be breaking.
These are green or red in Chrome/Safari/...

Ours is blue before and after, just a slightly different tint.